examples/bucket_setup_using_autokey/outputs.tf

```
@@ -14,17 +14,12 @@
  * limitations under the License.
  */
 
-output "autokey_config_id" {
+output "bucket_keyhandle" {
   description = "An Autokey configuration identifier."
-  value       = module.autokey.autokey_config_id != null ? module.autokey.autokey_config_id : ""
+  value       = resource.google_kms_key_handle.bucket_keyhandle
 }
 
-output "autokey_keyhandles" {
+output "bucket_name" {
   description = "A map of KeyHandles created."
```

description needs to be updated
```suggestion
  description = "The resource name of the storage bucket."
```

---

done